### ISSUE TITLE
Add Tabs component

### ISSUE THREAD

**Issue by RobjS** [dxw/govuk-components-plugin#8]
https://design-system.service.gov.uk/components/tabs/

**Comment by dxw-serena** [comment]
I'm not sure if it is a good idea to add this component, because, as explained into the documentation:

> Do not use the tabs component if the total amount of content the tabs contain will make the page slow to load. For this reason, do not use the tabs component as a form of page navigation.

I think that any client would use it in this wrong way as a first choice.
Open to discussion of course :)

**Comment by RobjS** [comment]
That's a good point. Hopefully the fact that the tabs can only be included in the main content will help to deter this (i.e. it can't be used to replace the main nav). But how about we include some kind of warning in the help text for the component?

**Comment by dxw-serena** [comment]
Good idea, and probably yes, the fact the it's in the main content should help deter users to abuse it.
